tests/slow/src/test/scala/tests/feature/ShowTastyLspSuite.scala

```diff
@@ -18,14 +19,62 @@ class ShowTastyLspSuite extends BaseLspSuite("showTasty") {
             |}
             |/app/src/main/scala/a/b/Main.scala
             |package a.b
-            |object Main {
-            |  println(5)
-            |}
+            |object Main
             |""".stripMargin
       )
       _ <- server.didOpen("app/src/main/scala/a/b/Main.scala")
       result <- server.showTasty(
-        s"$workspace/app/src/main/scala/a/b/Main.scala"
+        s"$workspace/app/src/main/scala/a/b/Main.scala",
+        new l.Position(1, 1)
+      )
+    } yield
+      if (result.isLeft) Assertions.fail("Command shouldn't failed")
+      else ()
+  }
+
+  test("handle-package-structure") {
+    for {
+      _ <- initialize(
+        s"""|/metals.json
+            |{
+            |  "app": {
+            |    "scalaVersion": "${V.scala3}"
+            |  }
+            |}
+            |/app/src/main/scala/Main.scala
+            |package foo.bar.example
+            |object Main
+            |""".stripMargin
+      )
+      _ <- server.didOpen("app/src/main/scala/Main.scala")
+      result <- server.showTasty(
+        s"$workspace/app/src/main/scala/Main.scala",
+        new l.Position(1, 1)
+      )
+    } yield
+      if (result.isLeft) Assertions.fail("Command shouldn't failed")
+      else ()
+  }
+
+  test("handle-multiple-classes") {
+    for {
+      _ <- initialize(
+        s"""|/metals.json
+            |{
+            |  "app": {
+            |    "scalaVersion": "${V.scala3}"
+            |  }
+            |}
+            |/app/src/main/scala/Main.scala
+            |package foo.bar.example
+            |class Foo
+            |class Bar
+            |""".stripMargin
+      )
+      _ <- server.didOpen("app/src/main/scala/Main.scala")
+      result <- server.showTasty(
+        s"$workspace/app/src/main/scala/Main.scala",
```

Comment:
The test doesn't really show what should be opened in this case.

Reply:
Yeah, but previously it would fail. I'm still thinking what will be the best (the fastest) way to tests all of decompile features